File: Data Structures & Algorithms/remove-duplicates-from-sorted-array-ii/submission-1.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        if (nums.size() == 0) return 0;
        int i = 2;
        int j = 2;

        while(i < nums.size()){
            if(nums[i] != nums[j-2]){
                nums[j] = nums[i];
                j++;
            }
            i++;
        } 
        return j;
    }
};

File: Data Structures & Algorithms/minimum-size-subarray-sum/submission-1.java
class Solution {
    public int minSubArrayLen(int target, int[] nums) {
        
        int L = 0;
        int currSum = 0;
        int length = Integer.MAX_VALUE;
        for (int R = 0; R < nums.length; R++){
            if (R <= nums.length - 1){
                currSum += nums[R];
                while(currSum >= target){
                length = Math.min( R-L+1 , length);
                currSum -= nums[L];
                L++;
                }
            }
        }
        if (length == Integer.MAX_VALUE){
            return 0;
        }
        return length;
    }
};